terraform/testcases/standard_otlp_metric_trace/parameters.tfvars

```diff
@@ -0,0 +1,3 @@
+validation_config = "standard-otel-trace-metric-validation.yml"
+
+soaking_data_mode = "trace"
```

Comment:
What is this used for? Are we planning on doing soaking tests for this test case?